File: Introducao/introducao.js
class Program {
    x = 1;
    y = 2;

    introducao = function() {
        alert('O resultado da soma é: ' + (x + y));
    }
    introducao;
}

chama = new Program();
chama.introducao;

//Classes 
Carro = class {
    constructor(){
        this.roda = "";
        this.volante = "";
        this.painel = "";
    }

    andar (){
        alert(this.nome);
    }

    parar (){
        alert(this.telefone);
    }

    static todosOsCarros() {
        return itens;
    }
    static itens = [];
}

File: Introducao/introducao.cs
using System;

namespace OrientacaoObjetosIntroducao
{
    //Classes
    public class Carro
    {
        public Carro()
        {

        }

        private string vprivate = "";

        public string roda {
            get { return "Jhon"; }
            set { vprivate = value + "Zlatan"}
        }
        this.volante = "";
        this.painel = "";
        
        static void Main(string[] args)
        {
            int x = 1;
            int y = 2;
            Console.WriteLine("O resultado da operação é: " + (x + y));
        }
    }
    
}